v8go.cc

```diff
@@ -134,16 +134,38 @@ IsolateHStatistics IsolationGetHeapStatistics(IsolatePtr ptr) {
 
 /********** Context **********/
 
+// Print provides a basic interface to print to stdout.
+static void Print(const FunctionCallbackInfo<Value>& args) {
+  bool first = true;
+  auto iso = args.GetIsolate();
+  for (int i = 0; i < args.Length(); i++) {
+    HandleScope handle_scope(iso);
+    if (first) {
+      first = false;
+    } else {
+      printf(" ");
```

Comment:
`printf` and `println` builtins are  not designed for production code:

> Print is useful for bootstrapping and debugging; it is not guaranteed to stay in the language. 

Would be better if we could find an API that uses an `io.Writer` so that you can set to `os.Stdout` or maybe `os.Stderr` but also any other file you want to "print" too.
https://tip.golang.org/pkg/builtin/#print